content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -25,6 +25,13 @@ further_reading:
   - .NET Core 3.1
   - .NET Core 2.1
 
+- .NET Tracer supports the above runtimes on the following processor architectures
```

Comment:
It's a bit weird to precise "the above runtimes", then for ARM "only .NET 5".

Maybe we can just say ".NET Tracer supports the following processor architectures" (with "the above runtimes" being implicit) ?

Reply:
done in https://github.com/DataDog/documentation/pull/10719/commits/5d66faa5bd2d0020a575eb0674a1c7f954efedfb